### ISSUE TITLE
Rule operator_usage_whitespace does not trigger for as? and is operators

### ISSUE THREAD

**Issue by ondrejhanslik** [realm/SwiftLint#3930]
Swiflint 0.47.0

### Environment

Swiflint 0.47.0 (Homebrew)

```yml
opt_in_rules:
  - operator_usage_whitespace
```
```swift
// This does not trigger a violation
let myProtocol = output   as     MyProtocol
```

**Comment by marcelofabri** [comment]
This is expected - when we talk about operators we are talking mostly about mathematical operators. `as` and `is` are considered other language constructs.

I'll keep this open though as a rule request.
